llm menu release note (#8079)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

Release notes are grouped by month for both multi-tenant and virtual private cloud (VPC) environments.

## October 2025

> - **New**: The docs.getdbt.com documentation site has introduced an LLM Context menu on all product documentation and guide pages. This menu provides users with quick options to interact with the current page using LLMs. You can can now:

Reviewer: ```suggestion
- **New**: The [docs.getdbt.com](http://docs.getdbt.com/) documentation site has introduced an LLM Context menu on all product documentation and guide pages. This menu provides users with quick options to interact with the current page using LLMs. You can can now:
```

>   - Copy the page as raw Markdown: making it easier to reference or reuse documentation content.

Reviewer: ```suggestion
  - Copy the page as raw Markdown &mdash; This makes it easier to reference or reuse documentation content.
```

>   - Open the page directly in ChatGPT or Claude: Automatically sharing the page’s content with the LLM, allowing you to start a chat with full page context preloaded.

Reviewer: Clicking this doesn't automatically share the content with the LLM. Instead, I get a pre-filled prompt like: "Read from https://docs.getdbt.com/xxx so I can ask questions about it." 

```suggestion
  - Open the page directly in ChatGPT or Claude &mdash; This redirects you to a chat with the LLM and automatically loads a message asking it to read the page, helping you start a conversation with context from the page.
```

>   <Lightbox src="/img/docs/dbt-cloud/llm-menu.png" width="80%" title="LLM Context menu on documentation pages" />

Author: ```suggestion
  <Lightbox src="/img/llm-menu.png" width="80%" title="LLM Context menu on documentation pages" />
```

>   <Lightbox src="/img/llm-menu.png" width="80%" title="LLM Context menu on documentation pages" />

Author: ```suggestion
  <Lightbox src="/img/llm-menu.png" width="50%" title="LLM Context menu on documentation pages" />
```
